tests.py

```diff
@@ -0,0 +1,87 @@
+import os
+import unittest
+import json
+import requests
+
+from webapi.routes import create_app
+from webapi.api_data import Users, Events
+
+class TestAPIs(unittest.TestCase):
+    """Contains all tests"""
+    def setUp(self):
+        self.app = create_app(config_name="testing")
+        self.app.config['SECRET_KEY'] = 'my-secret-key'
+        with self.app.app_context():
+            pass
+
+    def test_register_json(self):
+        tester = self.app.test_client(self)
+        response = tester.post('/api/v2/auth/register',
+                               data=dict(username = "admin", password = "1234"))
+        self.assertEqual(response.status_code, 201, msg="Register api not working")
```

Comment:
your status code says created, the message indicates the api not working, could you kindly resolve this

Reply:
Okay.